docs/networks/add-network.md

```diff
@@ -0,0 +1,26 @@
+---
+draft: true
+title: Add a Network
+intro: By default, every cloud provider provisions at least one network/VPC (Virtual Private Cloud) in each region to which a user's account has access. However there are cases where you might need to define custom network topology, for example to isolate resources that should not be directly connected. Add a network to define a custom network topology.
```

Comment:
As we support multiple cloud providers, it's too risky to use the `intro` of a topic to try to explain how things work in cloud providers in general.
The assumption made in previous text that "every cloud provider ... at least one VPC ..." is too bold, as some CSPs provision network per region, others do it globally, other CSPs that we still do not support might adopt a different approach.

Please let's just keep things simple, as we've been trying in most topics of our documentation.

Let's go straight to the point, say what a user can do, and perhaps a quick example of when that could be helpful. And that's it. Intro texts should be short.

```suggestion
intro: Add a network to your environment when you need to define custom network topology, so you can have resources (such as Servers and Applications) deployed to different networks, will be completely isolated from each other.
```